web3/method.py

```diff
@@ -40,8 +40,6 @@
     from web3 import Web3  # noqa: F401
     from web3.module import Module  # noqa: F401
 
-Munger = Callable[..., Any]
```

Comment:
Is this change just to simplify it a bit since this is pretty general anyhow?

Reply:
oops, I don't remember why I did this. Will put it back!